enunion-helper/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+use std::fmt::Write as _;
+use std::io::Write;
+use std::os::windows::process::ExitStatusExt;
+use std::path::PathBuf;
+use std::process::ExitStatus;
+
+/// Post-build helper event
+///
+/// This function appends the generated TypeScript code with enums
+/// from enunion to index.d.ts, which is generated by the napi library.
+/// The folder enunion-generated-ts and index.d.ts must be in one directory.
+/// For TypeScript only.
+pub async fn post_build(enunion_target_path: PathBuf) -> Result<(), ExitStatus> {
+    let build_dir = PathBuf::from(".").join(enunion_target_path);
+    let enunion_path = build_dir.join("enunion-generated-ts");
+    let ts_file_path = build_dir.join("index.d.ts");
```

Comment:
- [ ] please make these configurable, assuming "." or the name of the output file etc is not helpful